pkgs/development/python-modules/py-datastruct/default.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
```

```suggestion
  poetry-core,
```
Inside `python-modules`, call dependencies normally. The `callPackage` in `python-packages.nix` is modified to also include python dependencies in the top-level.